app/views/videos/advanced_search.html.haml

```diff
@@ -0,0 +1,50 @@
+%section.advanced_search.container
+  = form_tag(advanced_search_videos_path, method: 'get', class: "form-horizontal") do
+    %header
+      %h1 Advanced Search
+    .form-group
+      .col-sm-6
+        = search_field_tag :query, params[:query], placeholder: 'Search videos by title, description', class: 'form-control'
+    .form-group
+      .col-sm-6
+        .checkbox
+          %label
+            = check_box_tag "reviews", false
+            Include Reviews
+    .form-group
+      .row.average_rating_label Average Rating
+      %label.col-sm-1.control-label(for="rating_from") From:
+      .col-sm-2
+        = select_tag :rating_from, options_for_ratings_filter, prompt: '-', class: 'form-control'
+      %label.col-sm-1.control-label(for="rating_to") To:
+      .col-sm-2
+        = select_tag :rating_to, options_for_ratings_filter, prompt: '-', class: 'form-control'
+    .form-group
+      .col-sm-6
+        = button_tag(type: "submit", class: "btn btn-primary") do
+          %span.glyphicon.glyphicon-search
+          Search
+  %hr
+  - if @videos != []
```

Comment:
`@videos != []` means not empty? - double negation. I recommend `@videos.any?`, which is the opposite of `empty?`.